### ISSUE TITLE
[BUG]: accessibility of combo boxes, screen reader doesn't read items on selection

### ISSUE THREAD

**Issue by LeonarddeR** [microsoft/MIDI#1223]
Note, since this is an issue regarding UI Accessibility, I stripped unrelated stuff from the template.

### Report pre-requisites

- [x] I have searched this repo for existing issues to confirm this is not a duplicate
- [x] I agree to follow this project's Code of Conduct
- [x] I acknowledge that if I do not respond to follow-up questions in a timely manner, the issue may be closed without any changes or fixes

### Clarity and Disclosure

- [x] I confirm that any AI analysis included in this report is clearly called out / tagged as AI-generated with "AI Generated Content". Reports with suspected AI content may be closed if not disclosed.
- [x] I confirm that I will quickly follow-up to questions or requests for additional detail, otherwise the issue may be closed

### Windows Version

Windows 11 pro 26H2 26300.9539 release preview

### Type of bug

- [x] Application not working as expected

### Location

- [x] Windows MIDI Services tools like MIDI Settings, Console, etc.

### Application Name and Version

Windows Midi Player from preview 9

### Your personal observation that led to the bug report

I was testing the new Midi Player application with screen reading software (e.g. [NVDA](https://www.nvaccess.org/) and Windows built-in narrator)

### Steps to reproduce

1. Make sure a screen reader is active (e.g. Windows Narrator)
2. Open the Windows MIDI Player app
3. Tab to the "Play to" combo box

#### Expected behavior:

1. Screen reader reads the selected end point (e.g. General MIDI Synth) after the combo box label and role.
2. Changing the selection in the combo box with up/down arrow reads the selected endpoint name

### Actual behavior:

1. Narrator only announces "play to, combo box, collapsed"
2. Changing the selection in the combo box only reports the position in the list, e.g. 1 of 3, 2 of 3, etc.

### Additional notes

The combo box has the UI automation id of `EndpointComboBox` and a class name of `ComboBox`.
The combo box items have a class name of `ComboBoxItem`. Their Name property emit an empty value. They have a child node with class name `TextBlock` that properly exposes the name. The name should also be propagated to the combo box items.

This issue does not stand on its own, i.e. it also applies to the list of tracks, the list of items in the queue. The items in the list are not announced because the track name isn't exposes as a name on the `ListViewItem` UIA Elements.
An example of proper combo box authoring can be found in the MIDI Settings app. There, combo boxes like "Show endpoints from" read fine. However, the midi endpoints list in the MIDI Settings app suffers from the same unlabeled issue. I'm happy to provide a similar issue for this if desirable.

**Comment by Psychlist1972** [comment]
Thanks Leonard. We will investigate and fix across the whole set of apps.

Pete
Microsoft

**Comment by Psychlist1972** [comment]
Updated Preview 9 with fixes to this. If you find any others, please do let me know. Feel free to reopen this bug and add comments if related.

Pete
Microsoft

**Comment by LeonarddeR** [comment]
Many thanks @Psychlist1972, this works like a charm!
